src/components/SettingsDialog/Components/SettingsDropdown.tsx

```diff
@@ -0,0 +1,70 @@
+import {FC, useState} from "react";
+import "./SettingsDropdown.scss";
+import {ReactComponent as DropdownIcon} from "assets/icon-arrow-next.svg";
+import {ReactComponent as German} from "assets/flags/DE.svg";
+import {ReactComponent as English} from "assets/flags/US.svg";
+import {useTranslation} from "react-i18next";
+import i18n from "i18n";
+
+interface SettingsDropdownProps {
+  showDropdown?: boolean;
+}
+
+export const SettingsDropdown: FC<SettingsDropdownProps> = ({showDropdown}) => {
```

Comment:
This component should also use `VFC` because it doesn't receive children

Reply:
Maybe call the component `LanguageSettingsDropdown` so everyone knows that this is specifically for changing the language.